thentos-core/purescript/.gitignore

```diff
@@ -0,0 +1,5 @@
+/bower_components/
```

Comment:
I think this was created by pulp (?), but we may want to stick to a single .gitignore for the whole project?